### PR TITLE
Add meteostat back fill

### PR DESCRIPTION
**Pull request by ldodds**
Add a new cron job to back fill Meteostat data.

### CODE REVIEW

File: .ebextensions/cronjob.config [hunk 1]
```diff
@@ -28,15 +28,22 @@ files:
         owner: root
         group: root
         content: |
-            45 3 * * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bin/run_as_webapp bundle exec rake data_feeds:meteostat_loader >> log/jobs.log 2>&1'
+            50 3 * * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bin/run_as_webapp bundle exec rake data_feeds:meteostat_loader >> log/jobs.log 2>&1'
 
-    "/etc/cron.d/back-fill-any-dark-sky-temperatures":
+    "/etc/cron.d/back-fill-any-meteostat-temperatures":
```
Comment (jhigman) [inline .ebextensions/cronjob.config:33]: Are these jobs named the right way round (meteostat and dark sky backfills)..?

Comment (ldodds) [reply .ebextensions/cronjob.config]: d'oh, edited the wrong line

**Comment by ldodds** [comment]
Fixed the naming error

**Comment by jhigman** [comment]
👍

**Pull request merged**